plugins/default/webhook.ts

```diff
@@ -20,7 +20,11 @@ export const handler: PluginHandler = async (
   let data = null;
   try {
     let url = parameters.webhookURL;
-    const headers = parseHeaders(parameters.headers);
+
+    let headers: Record<string, string> = parameters?.headers
```

Comment:
Hey. Looks like this got changed while resolving some merge conflicts? We can keep the existing `parseHeaders` function call because it handles both: object and stringified headers.

Reply:
Damn, sorry about this! Reverted it back